### PR TITLE
Add executablePath to options

### PR DESCRIPTION
**Pull request by linonetwo**
To support https://github.com/stereobooster/react-snap/issues/93 , add a single line.

### CODE REVIEW

**Comment by stereobooster** [comment]
```js
const defaultOptions = {
  ...
  headless: true,
  puppeteerArgs: [],
  puppeteerExecutablePath: undefined,
  ...
}
```

**Comment by linonetwo** [comment]
@stereobooster  Done.

File: src/puppeteer_utils.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ const crawl = async opt => {
   const browser = await puppeteer.launch({
     headless: options.headless,
     args: options.puppeteerArgs,
+    executablePath: options.executablePath,
```
Comment (stereobooster) [inline src/puppeteer_utils.js]: executablePath: options.puppeteerExecutablePath

Comment (linonetwo) [reply src/puppeteer_utils.js]: Oh...Silly me.

**Comment by stereobooster** [comment]
I can merge as is, but it would be nice to have some documentation

**Comment by linonetwo** [comment]
What do you think, my English might be poor...

File: Readme.md [hunk 2]
```diff
@@ -97,6 +97,12 @@ Use `inlineCss: true` to enable this feature.
 
 TODO: as soon as the feature will be stable it should be enabled by default.
 
+### puppeteerExecutablePath
```
Comment (stereobooster) [inline Readme.md]: Name section `Docker` and move under `Containers and other restricted environments`.

Comment (linonetwo) [reply Readme.md]: Reasonable.

**Comment by stereobooster** [comment]
Thank you. You do not need to be sorry for your english or forgetting something. Any contribution is valuable if it improves functionality. 👍

**Pull request merged**